test/headers-test.ts

```diff
@@ -0,0 +1,60 @@
+// /test/headers-test.ts
+// Tests whether the headers sent back by the middleware
+
+import request from 'supertest'
+
+import rateLimit from '../dist/esm/index.js'
+
+import { createServer } from './helpers/create-server.js'
+
+describe('headers test', () => {
+	it('should send correct `x-ratelimit-limit`, `x-ratelimit-remaining`, and `x-ratelimit-reset` headers', async () => {
+		const app = createServer(
+			rateLimit({
+				windowMs: 60 * 1000,
+				max: 5,
+				legacyHeaders: true,
+			}),
+		)
+		const expectedResetTimestamp = Math.ceil(
+			(Date.now() + 60 * 1000) / 1000,
+		).toString()
+		const resetRegexp = new RegExp(`^\\d+${expectedResetTimestamp.slice(-2)}$`)
```

Comment:
Previously this had `\d\d` at the end, now it has `\d+` at the beginning - why did that change?

The intention was to match the timestamp, but ignore the last two digits to allow for a bit of wiggle room.

Reply:
Oh, right, slice with a negative 2 gets just the last two characters:

```js
'1234567890'.slice(-2)
// > "90"
```

So, kind of the opposite of the goal here.

```suggestion
		const resetRegexp = new RegExp(`^${expectedResetTimestamp.substr(0, expectedResetTimestamp.length - 2)}\\d\\d$`)
```